### PR TITLE
Fix newlines for Android

### LINKED ISSUE #480: No linebreaks in email body on Android Gmail
The Android Gmail client app doesn't seem to respect the line breaks added in your emails when opened via the "Send email" link.

See https://imgur.com/IYJQGKG

This issue doesn't seem to reproduce on other 3rd party email apps (tested on Spark, Outlook). Perhaps Gmail needs explicit carriage returns?

### PR DESCRIPTION
**Pull request by dionlarson**
Closes #480.

## QA

Needs QA on Android and iOS.

Check any email to see if newlines are respected in the email client (`send email` link).

I set up a [subdomain](https://defund.dionlarson.com/) for QA so you don't need to set up anything locally! ❤️

### CODE REVIEW

**Comment by dionlarson** [comment]
Closes #480.

**Comment by dionlarson** [comment]
I have QA'd on...
✅ Chrome, macOS
✅ Android 10, Chrome, Gmail
✅ iOS 13, Safari, Mail
🚫 iOS, Gmail (but this also does not work on production)

File: _layouts/email.html [hunk 1]
```diff
@@ -42,8 +42,13 @@ <h2>{{ page.name }}</h2>
 
 <script type="text/javascript">
     function openEmail() {
+        const isAndroid = /(android)/i.test(navigator.userAgent);
         const subject = encodeURIComponent(`{{ site.default_subject_line }}`.trim());
-        const body = encodeURIComponent(`{{ page.body }}`.trim());
+        const body = encodeURIComponent(
+            isAndroid ?
```
Comment (dionlarson) [inline _layouts/email.html:48]: Without this check...
🚫 `<br>` shows up on macOS / Gmail
🚫 Double newlines (but no `<br>`) on iOS / Mail
🚫 No newlines and `<br>` shows up on iOS / Gmail

**Comment by dionlarson** [comment]
@bingbongle [try this](mailto:testing@email.com;testing2@email.com?subject=Character Return Test&body=abcdefghijklmnopqrztuvwxyz%0D%0A0123456789%0D%0Aabcdefghijklmnopqrztuvwxyz%0D%0A0123456789) on iOS / Gmail

File: _layouts/email.html (hunk 1 above)
Comment (bingbongle) [reply _layouts/email.html:48]: Let's go ahead and merge so we can unblock android. Great work!

**Pull request merged**